src/utils/RunSUSHI.js

```diff
@@ -149,3 +153,40 @@ function printResults(pkg) {
   results.forEach((r) => console.log(r));
   // results.forEach((r) => console.log(`%c${r}`, `color:${clr}`)); // Color formatting for browser console
 }
+
+function printGoFSHresults(pkg) {
+  const proNum = pad(pkg.profiles.length.toString(), 12);
+  const extNum = pad(pkg.extensions.length.toString(), 13);
+  const vsNum = pad(pkg.valueSets.length.toString(), 12);
+  const csNum = pad(pkg.codeSystems.length.toString(), 13);
+  const instNum = pad(pkg.instances.length.toString(), 12);
+  const invNum = pad(pkg.invariants.length.toString(), 13);
+  const mapNum = pad(pkg.mappings.length.toString(), 12);
+  const errNumMsg = pad(`${stats.numError} Error${stats.numError !== 1 ? 's' : ''}`, 12);
+  const wrnNumMsg = padStart(`${stats.numWarn} Warning${stats.numWarn !== 1 ? 's' : ''}`, 12);
+  const creatures = pad(getRandomSeaCreatures(), 13);
+  const creatrStat = pad(getRandomSeaCreaturesStat(stats.numError, stats.numWarn), 13);
+  const aWittyMessageInvolvingABadFishPun = padEnd(getRandomPun(stats.numError, stats.numWarn), 37);
+
+  // prettier-ignore
+  const results = [
+    '╔'  + '═════════════════════════ GoFSH RESULTS ═════════════════════════' +'╗',
+    '║' + ' ╭──────────────┬───────────────┬──────────────┬───────────────╮ ' + '║',
+    '║' + ' │   Profiles   │  Extensions   │  ValueSets   │  CodeSystems  │ ' + '║',
+    '║' + ' ├──────────────┼───────────────┼──────────────┼───────────────┤ ' + '║',
+    '║' + ` │ ${ proNum  } │ ${  extNum  } │ ${  vsNum  } │ ${  csNum   } │ ` + '║',
+    '║' + ' ╰──────────────┴───────────────┴──────────────┴───────────────╯ ' + '║',
+    '║' + ' ╭──────────────┬───────────────┬──────────────┬───────────────╮ ' + '║',
+    '║' + ` │  Instances   │  Invariants   │   Mappings   │ ${creatures } │ ` + '║',
+    '║' + ' ├──────────────┼───────────────┼──────────────┼───────────────┤ ' + '║',
+    '║' + ` │ ${ instNum } │ ${  invNum  } │ ${ mapNum  } │ ${creatrStat} │ ` + '║',
+    '║' + ' ╰──────────────┴───────────────┴──────────────┴───────────────╯ ' + '║',
+    '║' + '                                                                 ' + '║',
+    '╠'  + '═════════════════════════════════════════════════════════════════' +'╣',
+    '║' + ` ${aWittyMessageInvolvingABadFishPun } ${errNumMsg} ${wrnNumMsg} ` + '║',
+    '╚'  + '═════════════════════════════════════════════════════════════════' +'╝'
+  ];
+
+  console.log();
```

Comment:
I think this is trying to add a blank line before the box, but it doesn't look like it's getting added. The SUSHI one logs a `' '`, which I think should work here, if that was the intension.

Reply:
Fixed in https://github.com/FSHSchool/FSHOnline/pull/43/commits/dacbc2f697103e8ee0f3759fe0f6951a057de2fb